spread.yaml

```
@@ -11,6 +11,36 @@ exclude:
   - .github
 
 backends:
+  # We can switch back to using the official Spread LXD backend after
+  # https://github.com/snapcore/spread/pulls?q=is%3Apr+is%3Aopen+LXD+updated%3A%3C%3D2024-02-23+
+  lxd:
+    type: adhoc
+    allocate: |
+      set -x
+      release=$(echo $SPREAD_SYSTEM | awk -F '-' '{print $2}')
+
+      # Ideally, we would add the ubuntu-minimal remote
+      # e.g. https://cloud-images.ubuntu.com/minimal/releases/jammy/
+      # but that would effectively change the host's LXC configurations.
+
+      echo "Allocating $SPREAD_SYSTEM..."
+      lxc launch --ephemeral ubuntu:$release $SPREAD_SYSTEM
+      until lxc exec $SPREAD_SYSTEM -- systemctl status | grep "running"
+      do
+        sleep 5
+      done
+      lxc exec $SPREAD_SYSTEM -- systemctl enable --now ssh
+      lxc exec $SPREAD_SYSTEM -- sed -i 's/^\s*#\?\s*\(PermitRootLogin\|PasswordAuthentication\)\>.*/\1 yes/' /etc/ssh/sshd_config
+      lxc exec $SPREAD_SYSTEM -- bash -c "sed -i 's/^\s*\(PermitRootLogin\|PasswordAuthentication\)\>.*/# COMMENTED OUT BY SPREAD: \0/' /etc/ssh/sshd_config.d/* || true"
+      lxc exec $SPREAD_SYSTEM -- bash -c "test -d /etc/ssh/sshd_config.d && echo -e 'PermitRootLogin=yes\nPasswordAuthentication=yes' > /etc/ssh/sshd_config.d/00-spread.conf"
+      lxc exec $SPREAD_SYSTEM -- bash -c "echo root:${SPREAD_SYSTEM_PASSWORD} | chpasswd"
+      lxc exec $SPREAD_SYSTEM -- killall -HUP sshd
```

Seems more appropriate?
```suggestion
      lxc exec $SPREAD_SYSTEM -- systemctl restart ssh
```

---

here I'm actually reproducing the official spread backend behaviour: https://github.com/snapcore/spread/blob/ded9133cdbceaf01f8a1c9decf6ff9ea56e194d6/spread/lxd.go#L486

So I'd rather keep it that way